storage-account.tf

```diff
@@ -1,5 +1,83 @@
 // Shared and specialised Storage Accounts
 
+locals {
```

Comment:
why is this local defined here? This doesn't seem like a good use of locals to me.